upstash_vector/core/index_operations.py

```diff
@@ -189,6 +202,64 @@ def query(
             )
         ]
 
+    def query_many(
```

Comment:
"could we make the existing query method accept list of queries as well" asks @ogzhanolguncu

Reply:
I think it might make sense in js sdk, because the parameters is already a single object, so making it list[obj] is easier, but here, each parameter is separate. If I want that, i would need to add another parameter called queries or something, and that might look ugly.

Reply:
I understand, it feels natural in TypeScript but in Python it doesn't look good.

query many is fine by me

Reply:
I didn't really think this. Yeah, in TS we can alter the return types but in py it's not possible. Okay, lets keep this as is 🫡